qchem/pennylane_qchem/qchem/obs.py

```diff
@@ -270,15 +279,22 @@ def observable(me_table, init_term=0, mapping="jordan_wigner", wires=None):
 
     **Example**
 
-    >>> table = np.array([[0.0, 0.0, 0.4], [1.0, 1.0, -0.5], [1.0, 0.0, 0.0]])
-    >>> print(observable(table, init_term=1 / 4, mapping="bravyi_kitaev"))
-    (0.2) [I0]
-    + (-0.2) [Z0]
-    + (0.25) [Z0 Z1]
-    >>> print(observable(table, init_term=1 / 4, mapping="bravyi_kitaev", wires=['w0','w1']))
-    (0.2) [Iw0]
-    + (-0.2) [Zw0]
-    + (0.25) [Zw0 Zw1]
+    >>> t = np.array([[0., 0., 0.5], [1.0, 1.0, -0.5], [1.0, 0., 0.]])
+    >>> v = np.array([[ 0., 0., 0., 0., 0.25], [ 0., 1., 1., 0., -0.25], [ 1., 0., 0., 1., -0.5]])
+    >>> matrix_elements = []
+    >>> matrix_elements.append(t)
+    >>> matrix_elements.append(v)
+    >>> print(observable(matrix_elements, init_term=1/4, mapping="bravyi_kitaev"))
+    (0.15625) [I0]
+    + (-0.15625) [Z0]
+    + (0.34375) [Z0 Z1]
+    + (-0.09375) [Z1]
+
+    >>> print(observable(matrix_elements, init_term=1/4, mapping="bravyi_kitaev", wires=['w0','w1']))
```

Comment:
I think you should delete this. It's the same line as before, except that it has different wire labels, right? If you want an example with wire labels, I would suggest instead
```
observable(matrix_elements, init_term=1/4, mapping="bravyi_kitaev", wires=['a','b']))
```
and remove the print without labels